<!-- CPGR, 2020, author: seungwookim -->

<!DOCTYPE html>
<html lang="ko">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="initial-scale=1.0, maximum-scale=1.0, width=device-width, user-scalable=no" />

    <title>고려대학교 유전단백체 연구센터</title>

    <link rel="stylesheet" type="text/css" href="css/default.css">
    <link rel="stylesheet" type="text/css" href="css/index.css">
    <link rel="stylesheet" type="text/css" href="css/slideshow.css">
    <link rel="stylesheet" media="( min-width:701px ) and ( max-width:1000px )" type="text/css" href="css/tablet.css">
    <link rel="stylesheet" media="( min-width:0 ) and ( max-width:700px )" type="text/css" href="css/mobile.css">
    <link rel="stylesheet" href="http://cdn.jsdelivr.net/npm/xeicon@2.3.3/xeicon.min.css">
    <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/gh/moonspam/NanumBarunGothic@latest/nanumbarungothicsubset.css">

    <script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
    <script src="JS/default.js"></script>
    <script src="JS/slider.js"></script>
    <script src="JS/pagination.js"></script>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
    <script src="https://cdn.jsdelivr.net/bxslider/4.2.12/jquery.bxslider.min.js"></script>

</head>
<body>

    <!-- NAVBAR -->
    <header>
        <div class="logo">
            <h1><a href="../../">CPGR</a></h1>
            <div><a href="#" class="all">All menu</a></div>
        </div>
        <div class="menu">
            <div>
                <h2 class="ko">유전단백체 연구센터</h2>
                <h2 class="en">Center for ProteoGenome Research</h2>
                <nav>
                    <ul>
                        <li><a href="../../about">센터소개</a></li>
                        <li><a href="../../members">센터멤버</a></li>
                        <li><a href="../../facilities">센터시설</a></li>
                        <li><a href="../highlights">연구현황</a>
                            <ul>
                                <li><a href="../highlights">Research Highlights</a></li>
                                <li><a href="../publicationlist">Publication list</a></li>
                            </ul>
                        </li>
                        <li><a href="../../news">센터소식</a></li>
                    </ul>
                </nav>
                <a href="#" class="close">Close</a>
            </div>
        </div>
    </header>

    <div class="bigcontainer">
        <div class="contents">
            <div class="flexbox">
                <h2><a href="#">Research Highlights</a></h2>
                <h2><a href="../publicationlist">Publication List</a></h2>
            </div>

            <div class="list">
                <div id="page1">
                    <div>
                        <h3>Prognostic plasma protein panel for Aβ deposition in the brain in Alzheimer’s disease</h3>
                        <p>Park, J.-C.;
                            Han, S.-H.; Lee. H.; Jeong, H.; Byun, M. S.; Bae, J.; Kim, H.; Lee, D. Y.; Yi, D.; Shin, S. A.;
                            Kim, Y. K.; Hwang, D.; Lee, S.-W.; Mook-Jung, I. Progress in Neurobiology 2019, 183, 1-13</p>
                        <div class="flexbox">
                            <img src="imgsrc/figure/hl1.jpg" width="300px" height="200px">
                            <div class="description">
                                <p>Alzheimer’s disease (AD) is the most common age-associated dementia. Many studies have sought
                                    to predict cerebral amyloid deposition, the major pathological hallmark of AD, using body
                                    fluids such as blood or cerebral spinal fluid (CSF). The use of blood in diagnostic
                                    procedures is widespread in medicine; however, existing blood biomarkers for
                                    AD remain unreliable. We sought to discover blood biomarkers that discriminate
                                    Aβ deposition status in the brain. This study used 107 individuals who were
                                    cognitively normal (CN), ...</p>

                                <div>
                                    Learn more: <a href="https://doi.org/10.1016/j.pneurobio.2019.101690">
                                        https://doi.org/10.1016/j.pneurobio.2019.101690</a>
                                </div>
                            </div>
                        </div>



                    </div>

                    <div>
                        <h3>Accurate Precursor Mass Assignment Improves Peptide Identification in Data-Independent
                            Acquisition Mass Spectrometry</h3>
                        <p>Mun, D.-G.; Nam, D.; Kim, H.; Pandey, A.; Lee, S.-W. Analytical Chemistry 2019, 91, 8453-8460</p>
                        <div class="flexbox">
                            <img src="imgsrc/figure/hl2.gif" width="300px" height="230px">
                            <div class="description">
                                <p>Proteomics research today no longer simply seeks exhaustive protein identification;
                                    increasingly, it is also desirable to obtain robust, large-scale quantitative information.
                                    To accomplish this, data-independent acquisition (DIA) has emerged as a promising
                                    strategy largely owing to developments in advanced mass spectrometers and sophisticated
                                    data analysis methods. Nevertheless, the highly complex multiplexed MS/MS spectra produced
                                    by DIA remain challenging to interpret. Here, we present a novel strategy to analyze DIA
                                    data, based on unambiguous precursor mass assignment through the mPE-MMR (multiplexed
                                    post-experimental monoisotopic mass refinement) ... </p>

                                <div>
                                    Learn more:
                                    <a href="https://doi.org/10.1021/acs.analchem.9b01474">https://doi.org/10.1021/acs.analchem.9b01474</a>
                                </div>
                            </div>
                        </div>
                    </div>

                    <div>
                        <h3>Comprehensive proteome and phosphoproteome profiling shows negligible influence of RNAlater on
                            protein abundance and phosphorylation</h3>
                        <p>Bae, J.; Kim, S.-J.; Lee, S.-E.; Kwon, W.; Kim, H.;  Han, Y.; Jang, J.-Y.; Kim, M.-S.; Lee, S.-W.
                            Clinical Proteomics 2019, 16, 18.</p>
                        <div class="flexbox">
                            <img src="imgsrc/figure/hl3.webp" width="300px" height="350px">
                            <div class="description">
                                <p>Certain tumors such as pancreatic ductal adenocarcinoma (PDAC) are known to contain a variety
                                    of hydrolytic enzymes including RNases and proteases that may lead to degradation of RNA and
                                    proteins during sample processing. For such tumor tissues with RNA instability, RNAlater
                                    containing a high concentration of quaternary ammonium sulfates that denature RNA-hydrolyzing
                                    enzymes is often used to protect RNAs from hydrolysis. Although a few studies have been
                                    carried out to determine the effect of RNAlater on DNA and RNA, whether RNAlater influences
                                    the proteome and phosphoproteome is largely unknown. In this study we carried out a
                                    systematic and comprehensive analysis of the effect of RNAlater on the proteome and
                                    phosphoproteome using high-resolution mass spectrometry. ... </p>

                                <div>
                                    Learn more:
                                    <a href="https://doi.org/10.1186/s12014-019-9239-z">https://doi.org/10.1186/s12014-019-9239-z</a>
                                </div>
                            </div>
                        </div>
                    </div>

                    <div>
                        <h3>Proteogenomic Characterization of Human Early-Onset Gastric Cancer</h3>
                        <p> Mun, D.-G.; Bhin, J.; Kim, S.; Kim, H.; Jung, J. H.; Jung, Y.; Jang, Y. E.; Park, J. M.;
                            Kim, H.; Jung, Y.; Lee, H.; Bae, J.; Back, S.; Kim, S.-J.; Kim, J.; Park, H.; Li, H.;
                            Hwang, K.-B.; Park, Y. S.; Yook, J. H.; Kim, B. S.; Kwon, S. Y.; Ryu, S. W.; Park, D. Y.;
                            Jeon, T. Y.; Kim, D. H.; Lee, J.-H.; Han, S.-U.; Song, K. S.; Park, D.; Park, J. W.;
                            Rodriguez, H.; Kim, J. Lee, H.; Kim, K. P.; Yang, E. G.; Kim, H. K.; Paek, E.; Lee, S.;
                            Lee, S.-W.; Hwang, D. Cancer Cell 2019, 35, 111-124.</p>
                        <div class="flexbox">
                            <img src="imgsrc/figure/hl4.jpg" width="300px" height="300px">
                            <div class="description">
                                <p>We report proteogenomic analysis of diffuse gastric cancers (GCs) in young populations.
                                    Phosphoproteome data elucidated signaling pathways associated with somatic mutations based
                                    on mutation-phosphorylation correlations. Moreover, correlations between mRNA and protein
                                    abundances provided potential oncogenes and tumor suppressors associated with patient
                                    survival. Furthermore, integrated clustering of mRNA, protein, phosphorylation, and
                                    N-glycosylation data identified four subtypes of diffuse GCs. Distinguishing these
                                    subtypes was possible by proteomic data. Four subtypes were associated with proliferation,
                                    immune response, metabolism, and invasion, respectively; and. ...
                                </p>

                                <div>
                                    Learn more:
                                    <a href="https://doi.org/10.1016/j.ccell.2018.12.003">https://doi.org/10.1016/j.ccell.2018.12.003</a>
                                </div>
                            </div>
                        </div>
                    </div>

                    <div>
                        <h3>A mitochondrial proteome profile indicative of type 2 diabetes mellitus in skeletal muscles</h3>
                        <p> Chae S.; Kim, S.-J.; Koo, Y. D.; Lee, J. H.; Kim, H.; Ahn, B. Y.; Ha, Y.-C.; Kim, Y.-H.;
                            Jang, M. G.; Koo, K.-H.; Choi, S. H.; Lim, S.; Park, Y. J.; Jang, H .C.; Hwang, D.; Lee, S.-W.;
                            Park, K. S. Experimental & Molecular Medicine 2018, 50:129.</p>
                        <div class="flexbox">
                            <img src="imgsrc/figure/hl5.webp" width="300px" height="260px">
                            <div class="description">
                                <p>The pathogenesis of type 2 diabetes mellitus (T2DM) is closely associated with mitochondrial
                                    functions in insulin-responsive tissues. The mitochondrial proteome, compared with
                                    the mitochondrial genome, which only contains 37 genes in humans, can provide more
                                    comprehensive information for thousands of mitochondrial proteins regarding T2DM-associated
                                    mitochondrial functions. However, T2DM-associated protein signatures in insulin-responsive
                                    tissues are still unclear. Here, we performed extensive proteome profiling of mitochondria
                                    from skeletal muscles in nine T2DM patients and nine nondiabetic controls. A comparison of
                                    the mitochondrial proteomes identified 335 differentially expressed proteins (DEPs) between
                                    T2DM and nondiabetic... </p>

                                <div>
                                    Learn more: <a href="https://doi.org/10.1038/s12276-018-0154-6">
                                    https://doi.org/10.1038/s12276-018-0154-6</a>
                                </div>
                            </div>
                        </div>
                    </div>

                </div>

                <div id="page2" style="display: none;">

                    <div>
                        <h3>Multi-omics analysis identifies pathways and genes involved in diffuse-type gastric
                            carcinogenesis induced by E-cadherin, p53, and Smad4 loss in mice</h3>
                        <p> Park, J. W.; Kim, M. -S.; Voon, D. C.; Kim, S.-J.; Bae, J.; Mun, D.-G.; Ko, S.-I.;
                            Kim, H. K.; Lee, S.-W.; Kim, D.-Y. Molecular Carcinogenesis 2018, 57, 947-954.</p>
                        <div class="flexbox">
                            <img src="imgsrc/figure/hl6.png" width="300px" height="300px">
                            <div class="description">
                                <p>The molecular mechanisms underlying the pathogenesis of diffuse‐type gastric cancer (DGC)
                                    have not been adequately explored due to a scarcity of appropriate animal models.
                                    A recently developed tool well suited for this line of investigation is the
                                    Pdx‐1‐Cre;Cdh1F/+ ;Trp53F/F;Smad4F/F (pChePS ) mouse model that spontaneously
                                    develops metastatic DGC showing nearly complete E‐cadherin loss. Here, we performed
                                    a proteogenomic analysis to uncover the molecular changes induced by the concurrent
                                    targeting of E‐cadherin, p53, and Smad4 loss. The gene expression profiles of mouse
                                    DGCs and in vivo gastric phenotypes from various combinations of gene knockout
                                    demonstrated that these mutations collaborate ...  </p>

                                <div>
                                    Learn more: <a href="https://doi.org/10.1038/s12276-018-0154-6">
                                    https://doi.org/10.1038/s12276-018-0154-6</a>
                                </div>
                            </div>
                        </div>
                    </div>

                    <div>
                        <h3>Comprehensive and sensitive proteogenomics data analysis strategy based on coplementary
                            multi-stage database search</h3>
                        <p> Madar, I. H.; Lee, Y.; Wang, X.; Ko, S.-I.; Kim, H.; Mun, D.-G.; Zhang B.; Paek, E.;
                            Lee, S.-W. International Journal of Mass Spectrometry 2018, 427, 11-19.</p>
                        <div class="flexbox">
                            <img src="imgsrc/figure/hl7.jpg" width="300px" height="260px">
                            <div class="description">
                                <p>Proteogenomics provide opportunities for proteomic validation of gene structures,
                                    genomic alterations and functional relevance of novel findings obtained from genomic
                                    data analysis. However, for effective proteogenomic data integration, an extensive
                                    proteome profiling, approaching the gene coverage of genomics data, is critical.
                                    Here we developed a multi-stage database search method for comprehensive proteomics
                                    data analysis to complement whole transcriptome sequencing data. The method utilizes
                                    two complementary database search engines, MS-GF+ and MODa/MODi, in tandem. The MS/MS
                                    data were first subjected to MS-GF+ database search (1st stage search) and the
                                    unidentified MS/MS data from the 1st stage search were subsequently analyzed ... </p>

                                <div>
                                    Learn more: <a href="https://doi.org/10.1038/s12276-018-0154-6">
                                    https://doi.org/10.1038/s12276-018-0154-6</a>
                                </div>
                            </div>
                        </div>
                    </div>

                </div>
            </div>

            <div class="pagination">
                <div>
                    <a id="1" href="javascript: onClick=showPage('1')">1</a>
                    <a id="2" href="javascript: onClick=showPage('2')">2</a>
                </div>
            </div>

        </div>



    </div>

    <footer>
        <p class="line1">고려대학교 유전단백체연구센터, Center for ProteoGenome Research (센터장: 이 상 원)</p>
        <p class="line2">(02841) 서울시 성북구 안암로 145 고려대학교 자연대캠퍼스 아산이학관 219호<br>Tel : 02-3290-3730 | E-mail : sw_lee@korea.ac.kr</p>
        <p class="line3">© 2020 고려대학교 유전단백체연구센터 (Center for ProteoGenome Research), All Rights Reserved </p>
    </footer>

</body>
</html>